src/components/segmentedControl/segment.tsx

```diff
@@ -116,7 +118,11 @@ const Segment = React.memo((props: SegmentProps) => {
     >
       {!iconOnRight && renderIcon()}
       {label && (
-        <Reanimated.Text fsTagName={'unmasked'} numberOfLines={1} style={[animatedTextStyle, Typography.text90]}>
+        <Reanimated.Text
+          fsTagName={'unmasked'}
+          numberOfLines={1}
+          style={[animatedTextStyle, Typography.text90, segmentLabelStyle]}
```

Comment:
I think it's better to set the animatedTextStyle at the end so it won't get overridden by the user's style..